src/util/Logger.ts

```diff
@@ -26,13 +26,18 @@ export class Logger {
    * Register log handler that logs to console.
    */
   static addConsoleHandler = (): void => {
+    const createHandler =
+      (level: Exclude<LogLevel, 'debug2'>): LogLevelHandler =>
+      (...args) =>
+        /* eslint-disable no-console */
+        console[level]('[vscode-deephaven]', ...args);
```

Comment:
You still want to use `.bind` - otherwise you lose debugging information. Most notably in console output, the line of code will always be this line (Logger.ts:33) instead of the actual line of code where you're logging from.
It's also a little more elegant, I think.
```suggestion
      console[level].bind(console, '[vscode-deephaven]');
```

Reply:
Good catch.